src/components/Thanks/OptInExpVariation.vue

```diff
@@ -0,0 +1,417 @@
+			<div class="tw-bg-eco-green-3 tw-text-center tw-pt-4 md:tw-rounded-t">
+				<h1 class="tw-text-white">
+					Success!
+				</h1>
+				<p class="tw-text-subhead tw-text-white tw-mb-2 tw-px-3">
+					{{ headerCopy }}
+				</p>
+
+				<div class="borrower-container">
+					<div class="tw-absolute tw-w-full tw-h-full tw-z-docked" style="width: 240px;">
+						<animated-sparkles />
+					</div>
+
+					<div
+						class="tw-flex tw-justify-center tw-items-center tw-z-1"
+						:style="{height: '200px'}"
+					>
+						<borrower-image
+							v-for="loan, index in filteredLoans"
+							:key="loan.id"
+							class="borrower-image"
+							:class="{'main-loan' : index === 1 && loans.length === 3}"
+							:style="{
+								marginLeft: `${loans.length === 2 && index === 1
+									? (index * marginLeftWeight)
+									: (index - 1) * marginLeftWeight}rem`,
+								zIndex: `${index + 1}`,
+							}"
+							:alt="loan.name"
+							:aspect-ratio="1"
+							:default-image="{ width: 200, faceZoom: 30 }"
+							:hash="hash(loan)"
+							:images="[
+								{ width: 200, faceZoom: 30 },
+							]"
+						/>
+					</div>
+				</div>
+			</div>
+			<div class="tw-relative tw-flex tw-justify-center">
+				<div class="tw-bg-marigold-1 tw-absolute oval"></div>
+				<div
+					class="tw-bg-marigold-1 tw-w-full tw-px-3 tw-border-b tw-border-b-secondary tw-z-1"
+				>
+					<h4 class="tw-text-center tw-pt-1">
+						What to expect next:
+					</h4>
+					<opt-in-steps
+						class="tw-mb-5"
+						:weeks-to-repay="weeksToRepay"
+					/>
+					<div class="tw-flex tw-flex-col tw-gap-2 tw-pb-3">
+						<template v-if="userOptedOut && isGuest">
+							<h3 class="tw-text-center">
+								<!-- eslint-disable-next-line max-len -->
+								Want to hear how you’re impacting {{ borrowerName }}’s life and more ways to help people like them?
+							</h3>
+							<kv-button
+								v-kv-track-event="[
+									'thanks',
+									'click',
+									'accept-opt-in-request',
+								]"
+							>
+								Yes, keep me updated
+							</kv-button>
+							<kv-button
+								variant="ghost"
+								class="ghost-button"
+								v-kv-track-event="[
+									'thanks',
+									'click',
+									'reject-opt-in-request',
+								]"
+							>
+								No, I don’t want to receive updates
+							</kv-button>
+						</template>
+						<template v-else>
+							<kv-button
+								to="/portfolio"
+								v-kv-track-event="[
+									'thanks',
+									'click',
+									'go-to-my-kiva',
+								]"
+							>
+								Go to my kiva
+							</kv-button>
+						</template>
+					</div>
+				</div>
+			</div>
+			<div class="tw-py-5 tw-px-3">
+				<div class="tw-mb-2">
+					<!-- eslint-disable-next-line max-len -->
+					<div
+						v-if="userOptedOut && isGuest"
+						class="
+								tw-w-full tw-border tw-rounded
+								tw-flex tw-justify-between tw-cursor-pointer
+								tw-py-2 tw-px-3
+							"
+						@click="openCreateAccount = !openCreateAccount"
```

Comment:
For ease of migration to Vue 3, this probably needs to be `() => openCreateAccount = !openCreateAccount`. But please verify that it still works in Vue 2 after changing.